.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1726,9 +1357,13 @@ public class FineTuningClient {
         protected internal FineTuningClient(ClientPipeline pipeline, Uri endpoint, OpenAIClientOptions options);
         public virtual ClientPipeline Pipeline { get; }
         public virtual ClientResult CancelJob(string jobId, RequestOptions options);
+        public ClientResult<FineTuningJob> CancelJob(string jobId, CancellationToken cancellationToken = default);
         public virtual Task<ClientResult> CancelJobAsync(string jobId, RequestOptions options);
+        public Task<ClientResult<FineTuningJob>> CancelJobAsync(string jobId, CancellationToken cancellationToken = default);
         public virtual ClientResult CreateJob(BinaryContent content, RequestOptions options = null);
+        public ClientResult<FineTuningJob> CreateJob(string training_file, string model, RequestOptions options = null);
```

Comment:
`training_file` -> `trainingFile` or maybe `inputFilename`; see [.NET conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions) for the "method parameters use camelCase" and other idiomatic stipulations. The [Azure SDK .NET guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html) are really handy, too, though we don't necessary match 100% of that given this isn't fundamentally an Azure SDK library.

Also -- and this is minor -- what are your thoughts on the parameter ordering? I'd intuitively think of model before filename ("I want to create a job that fine-tunes gpt-35-turbo using the input from this file here") but the other way isn't illogical, either ("I want to use this input file here to create a fine-tuned version of gpt-35-turbo").

Reply:
Changed the order; renamed to trainingFile. Decided against inputFilename to stay closer to openai's docs and payload.